src/index.scss

```diff
@@ -1,3 +1,5 @@
+@import '../node_modules/uswds/dist/scss/uswds.scss';
+
 .uswds-thing {
```

Comment:
This class name does not seem to be super descriptive.  Was this just for testing?

Reply:
@xylocopaCaerulea yes `.uswds-thing` was just for the initial testing, I can remove it.